examples/elasticsearch/expose-disable.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: OpsRequest
+metadata:
+  name: es-expose-disable
+  namespace: default
+spec:
+  # Specifies the name of the Cluster resource that this operation is targeting.
+  clusterName: es-cluster
+  # Lists Expose objects, each specifying a Component and its services to be exposed.
+  expose:
+  # Specifies the name of the Component.
+  - componentName: mdit
+    # Specifies a list of OpsService. When an OpsService is exposed, a corresponding ClusterService will be added to `cluster.spec.services`.
+    services:
+    - name: internet
+      serviceType: LoadBalancer
+      ports:
+      - name: http
+        port: 9200
+        protocol: TCP
+        targetPort: http
```

Comment:
The port name has been renamed to `es-http`.

https://github.com/apecloud/kubeblocks-addons/blob/2f3687afa546200789eb5860d63fa16c56cae01f/addons/elasticsearch/templates/cmpd-8.yaml#L149-L155

Reply:
fixed